vendor/github.com/docker/distribution/testutil/handler.go

```diff
@@ -0,0 +1,148 @@
+package testutil
```

Comment:
The latter. I wasn't sure, which annotation is more appropriate. Is `<drop>` more fitting this case?